src/test/scala/com/sagacify/sonar/scala/MeasuresSpec.scala

```diff
@@ -48,51 +49,53 @@ class ScalaSensorSpec extends FlatSpec with Matchers {
 }
 """
 
+  private val scala211 = "2.11.11"
```

Comment:
In the future, we should definitely use ScalaCheck to verify the tests hold for all of the Scala versions supported by this plugin.